### DISCUSSION TITLE
File to compile

### DISCUSSION THREAD

**Discussion by mujojowang** [lum-ai/odinson#382] [Q&A]
How to get around this error
<img width="869" alt="スクリーンショット 2022-04-22 23 03 41" src="https://user-images.githubusercontent.com/96331731/164882299-e04a423c-c076-4b5f-871e-b35e8bbe6e9a.png">
?

**Comment by kwalcock** [comment]
The most recent published version of odinson is 0.6.1.  Is something requiring newer?  Perhaps you could explain what happened before these lines.  Are you running sbt from the main, top-level odinson directory?
